sympy/core/symbol.py

```diff
@@ -364,6 +364,13 @@ class Wild(Symbol):
     A Wild symbol matches anything, or anything
     without whatever is explicitly excluded.
 
+    Parameters
+    ==========
+
+    name : str
+    exclude : iterable
```

Comment:
How about we also mention here explicitly that `exclude` is optional, like this:
```
name : str
    a little explanation of its purpose
exclude : iterable, optional
    a little explanation of its purpose
properties : iterable of functions, optional
    a little explanation of its purpose
```

I can see that, it is mentioned below with examples. As how to use, `exclude`, `properties`, but I still think we can give here a very intro-level explanation. WDYT?

Reply:
Agreed, I updated accordingly.